app/models/node.rb

```diff
@@ -715,6 +722,23 @@ def self.new_note(params)
     [saved, node, revision]
   end
 
+  def self.new_preview_note(params)
+    author = User.find(params[:uid])
+    node = Node.new(uid:     author.uid,
+                    title:   params[:title],
+                    comment: 2,
+                    type:    'note')
+    revision = node.new_revision(uid:   author.uid,
```

Comment:
Is this working?
We have not saved `node ` so there would be no `id` for this node, so in `new_revision` method 'id' would be null, right?

https://github.com/publiclab/plots2/blob/fc11ad2b1b03aab6e3f313e8eab1d701cae49714/app/models/node.rb#L667

Reply:
Can you verify this?

Reply:
I've verified it, it creates the revision with both ids being nil
<img width="1100" alt="Screenshot 2020-10-21 at 17 22 15" src="https://user-images.githubusercontent.com/7622875/96733398-11507200-13c2-11eb-8249-88992a00b334.png">

Reply:
Cool!!!